split_train_valid.py

```diff
@@ -84,9 +85,15 @@ def main(args):
     print('Unique visitors in train set: ', len(visitors))
     print('Unique visitors in test set: ', len(visitors_test))
 
+    # p is the approximation of percent of dataset to split
+    p = args.percent_split/100
```

Comment:
import division from `__future__` to make sure this uses python3 true division even when running in python2. More details: https://www.python.org/dev/peps/pep-0238/#the-future-division-statement

Reply:
I have imported division from __future__